api/PerformanceElementTiming.json

```
@@ -339,6 +339,43 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "cross-origin": {
+          "__compat": {
+            "description": "Cross-origin",
```

Let's also add an MDN url here:

```suggestion
            "description": "Cross-origin",
            "mdn_url": "https://developer.mozilla.org/en-US/docs/Web/API/PerformanceElementTiming/renderTime#cross-origin_image_render_time",
```